topics/admin/tutorials/object-store/slides.html

```diff
@@ -15,211 +15,9 @@
   - natefoo
   - martenson
   - hexylena
----
-
-# Philosophy of Galaxy Storage
-
-- Foster transparency and reproducibility
-- Data is always created, *never overwritten*
-- Data is never deleted unless *explicitly instructed*
-- Even deleted data can be undeleted unless *forcibly purged*
-
-Additionally, tools can produce large amounts of transient data while running.
-
----
-
-class: normal
-# Mechanics
-
-* When files are produced they are stored in the filesystem defined by `file_path` config in `galaxy.yml`.
-  * Defaults to `database/files`.
-
---
-*  A database entry in `Dataset` table is created.
-  * Among others it has:  `| id	| deleted	| purged | file_size |`
-
---
-* To user we present the data through:
-  * Entries in `HistoryDatasetAssociation` table.
-    * Among others it has:  `| id	| history_id | deleted	| dataset_id |`
-  * Entries in `LibraryDatasetDatasetAssociation` table.
-    * Among others it has:  `| id	| library_dataset_id | deleted	| dataset_id |`
-
---
-Any number of 'Association' records can point to a base Dataset -- this is how copying histories, history items, and libraries work without needing to copy actual file contents.
-
----
-# Storage can grow fast
-
-An "average" NGS analysis (by Anton): **66 GB**
-
-10 users, 10 histories: **> 6 TB**
-
-Solutions:
-
-- Quotas
-- Clean up deleted data (with a cron job)
-- Forced removal based on age
-- Data libraries for common data
-
----
-# Data Libraries
-
-Provide a way to conveniently share Galaxy datasets within a group of Galaxy users or with everybody that has access to a specific instance of Galaxy.
-
-* Can import data from filesystem *without duplicating* it.
-
---
-* Can import *whole directories* preserving the folder structure.
-
---
-* The dataset's size does not count towards user's quota.
-  * Every dataset in the library is *stored only once* no matter how many users are using it in their histories.
-
---
-* Uses roles and groups to control permissions on library/dataset level.
-  * Only admins can _create_ libraries.
-  * Ordinary users can be granted permission to _manage_ libraries
-
----
-# Libraries configuration
-
-In `galaxy.yml`:
-
-* `user_library_import_dir`
-  * Allows authorized non-administrators to upload a directory of files.
-  * Directory must contain sub-directories named the same as user's email.
-  * Works well in combination with `ftp_upload_dir`.
-* `allow_path_paste`
-  * Admin-only, allows importing from any path that the Galaxy's user has access to.
-
-???
-If you use old library interface you can also set `library_import_dir`.
-It specifies which folder admins may browse and import from.
 
 ---
-# Libraries Exercise
```

Comment:
Did we capture this somewhere?

Reply:
Not yet. It's in history, but this wasn't an execrise anyone was running anyway. I'm sketching up  new tutorial.

Reply:
@natefoo These libraries slides have been moved to topics/admin/tutorials/stuff/slides.html for the moment.